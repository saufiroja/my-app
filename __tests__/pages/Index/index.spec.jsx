import React from 'react';
import renderer from 'react-test-renderer';
import Index from '../../../pages/index';

describe('Index', () => {
  it('matches the snapshot', () => {
    const tree = renderer.create(<Index />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
