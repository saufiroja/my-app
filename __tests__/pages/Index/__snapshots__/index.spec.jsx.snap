// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index matches the snapshot 1`] = `
Array [
  <div
    className="bg-hero bg-cover bg-primary h-[1050px]"
  >
    <div
      className="container mx-auto"
    >
      <div
        className="flex pt-16 items-center"
      >
        <div
          className="w-3/12"
        >
          <span
            style={
              Object {
                "background": "none",
                "border": 0,
                "boxSizing": "border-box",
                "display": "inline-block",
                "height": "initial",
                "margin": 0,
                "maxWidth": "100%",
                "opacity": 1,
                "overflow": "hidden",
                "padding": 0,
                "position": "relative",
                "width": "initial",
              }
            }
          >
            <span
              style={
                Object {
                  "background": "none",
                  "border": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": "initial",
                  "margin": 0,
                  "maxWidth": "100%",
                  "opacity": 1,
                  "padding": 0,
                  "width": "initial",
                }
              }
            >
              <img
                alt=""
                aria-hidden={true}
                src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27150%27%20height=%2726%27/%3e"
                style={
                  Object {
                    "background": "none",
                    "border": 0,
                    "display": "block",
                    "height": "initial",
                    "margin": 0,
                    "maxWidth": "100%",
                    "opacity": 1,
                    "padding": 0,
                    "width": "initial",
                  }
                }
              />
            </span>
            <img
              alt="impostor-logo"
              className="cursor-pointer"
              data-nimg="intrinsic"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style={
                Object {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": undefined,
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
            <noscript>
              <img
                alt="impostor-logo"
                className="cursor-pointer"
                data-nimg="intrinsic"
                decoding="async"
                loading="lazy"
                src="/_next/image?url=%2Fimages%2Flogo-white.png&w=384&q=75"
                srcSet="/_next/image?url=%2Fimages%2Flogo-white.png&w=256&q=75 1x, /_next/image?url=%2Fimages%2Flogo-white.png&w=384&q=75 2x"
                style={
                  Object {
                    "border": "none",
                    "bottom": 0,
                    "boxSizing": "border-box",
                    "display": "block",
                    "height": 0,
                    "left": 0,
                    "margin": "auto",
                    "maxHeight": "100%",
                    "maxWidth": "100%",
                    "minHeight": "100%",
                    "minWidth": "100%",
                    "objectFit": undefined,
                    "objectPosition": undefined,
                    "padding": 0,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                    "width": 0,
                  }
                }
              />
            </noscript>
          </span>
        </div>
        <div
          className="w-6/12"
        >
          <ul
            className="flex xl:-ml-[14%] lg:-ml-[14%] md:ml-[12%] sm:ml-[15%] xl:pb-[8px] lg:pb-[4px] md:pb-[7px] sm:pb-[6px] space-x-16"
          >
            <li>
              <a
                className="text-white text-lg cursor-pointer"
              >
                Home
              </a>
            </li>
            <li>
              <a
                className="text-white text-lg cursor-pointer"
                href="#about"
              >
                About
              </a>
            </li>
          </ul>
        </div>
        <div
          className="w-3/12 space-x-5 text-right"
        >
          <a
            className="text-lg py-3 px-10 rounded-2.5xl inline-block cursor-pointer bg-white text-primary"
            href="/login"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            Login
          </a>
          <a
            className="text-lg py-3 px-10 rounded-2.5xl inline-block cursor-pointer bg-white text-primary"
            href="/register"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            Register
          </a>
        </div>
      </div>
      <div
        className="w-5/12 pt-32"
      >
        <h1
          className="text-white text-8xl font-bold font-body pr-32"
        >
          let's show your talent for game
        </h1>
        <div
          className="ml-20"
        >
          <a
            className="xl:text-lg xl:py-3 xl:px-10 lg:text-lg lg:py-3 lg:px-8 md:px-6 md:text-md md:py-2.5 sm:px-6 sm:text-sm sm:py-2.5 rounded-2.5xl inline-block bg-yellow-500  mt-14"
          >
            Get Started
          </a>
        </div>
      </div>
    </div>
  </div>,
  <section
    className="pt-20"
  >
    <div
      className="container mx-auto"
    >
      <h2
        className="text-8xl font-bold font-body text-center"
      >
        List Game
      </h2>
      <div
        className="flex w-9/12 mx-auto mt-16"
      >
        <div
          className="w-4/12"
        >
          <a
            href="/login"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            <span
              style={
                Object {
                  "background": "none",
                  "border": 0,
                  "boxSizing": "border-box",
                  "display": "inline-block",
                  "height": "initial",
                  "margin": 0,
                  "maxWidth": "100%",
                  "opacity": 1,
                  "overflow": "hidden",
                  "padding": 0,
                  "position": "relative",
                  "width": "initial",
                }
              }
            >
              <span
                style={
                  Object {
                    "background": "none",
                    "border": 0,
                    "boxSizing": "border-box",
                    "display": "block",
                    "height": "initial",
                    "margin": 0,
                    "maxWidth": "100%",
                    "opacity": 1,
                    "padding": 0,
                    "width": "initial",
                  }
                }
              >
                <img
                  alt=""
                  aria-hidden={true}
                  src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27294%27%20height=%27217%27/%3e"
                  style={
                    Object {
                      "background": "none",
                      "border": 0,
                      "display": "block",
                      "height": "initial",
                      "margin": 0,
                      "maxWidth": "100%",
                      "opacity": 1,
                      "padding": 0,
                      "width": "initial",
                    }
                  }
                />
              </span>
              <img
                alt="game1"
                className="cursor-pointer"
                data-nimg="intrinsic"
                decoding="async"
                src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                style={
                  Object {
                    "border": "none",
                    "bottom": 0,
                    "boxSizing": "border-box",
                    "display": "block",
                    "height": 0,
                    "left": 0,
                    "margin": "auto",
                    "maxHeight": "100%",
                    "maxWidth": "100%",
                    "minHeight": "100%",
                    "minWidth": "100%",
                    "objectFit": undefined,
                    "objectPosition": undefined,
                    "padding": 0,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                    "width": 0,
                  }
                }
              />
              <noscript>
                <img
                  alt="game1"
                  className="cursor-pointer"
                  data-nimg="intrinsic"
                  decoding="async"
                  loading="lazy"
                  src="/_next/image?url=%2Fimages%2Fgame1.png&w=640&q=75"
                  srcSet="/_next/image?url=%2Fimages%2Fgame1.png&w=384&q=75 1x, /_next/image?url=%2Fimages%2Fgame1.png&w=640&q=75 2x"
                  style={
                    Object {
                      "border": "none",
                      "bottom": 0,
                      "boxSizing": "border-box",
                      "display": "block",
                      "height": 0,
                      "left": 0,
                      "margin": "auto",
                      "maxHeight": "100%",
                      "maxWidth": "100%",
                      "minHeight": "100%",
                      "minWidth": "100%",
                      "objectFit": undefined,
                      "objectPosition": undefined,
                      "padding": 0,
                      "position": "absolute",
                      "right": 0,
                      "top": 0,
                      "width": 0,
                    }
                  }
                />
              </noscript>
            </span>
          </a>
        </div>
        <div
          className="w-4/12"
        >
          <a
            href="/login"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            <span
              style={
                Object {
                  "background": "none",
                  "border": 0,
                  "boxSizing": "border-box",
                  "display": "inline-block",
                  "height": "initial",
                  "margin": 0,
                  "maxWidth": "100%",
                  "opacity": 1,
                  "overflow": "hidden",
                  "padding": 0,
                  "position": "relative",
                  "width": "initial",
                }
              }
            >
              <span
                style={
                  Object {
                    "background": "none",
                    "border": 0,
                    "boxSizing": "border-box",
                    "display": "block",
                    "height": "initial",
                    "margin": 0,
                    "maxWidth": "100%",
                    "opacity": 1,
                    "padding": 0,
                    "width": "initial",
                  }
                }
              >
                <img
                  alt=""
                  aria-hidden={true}
                  src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27294%27%20height=%27217%27/%3e"
                  style={
                    Object {
                      "background": "none",
                      "border": 0,
                      "display": "block",
                      "height": "initial",
                      "margin": 0,
                      "maxWidth": "100%",
                      "opacity": 1,
                      "padding": 0,
                      "width": "initial",
                    }
                  }
                />
              </span>
              <img
                alt="game2"
                className="cursor-pointer"
                data-nimg="intrinsic"
                decoding="async"
                src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                style={
                  Object {
                    "border": "none",
                    "bottom": 0,
                    "boxSizing": "border-box",
                    "display": "block",
                    "height": 0,
                    "left": 0,
                    "margin": "auto",
                    "maxHeight": "100%",
                    "maxWidth": "100%",
                    "minHeight": "100%",
                    "minWidth": "100%",
                    "objectFit": undefined,
                    "objectPosition": undefined,
                    "padding": 0,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                    "width": 0,
                  }
                }
              />
              <noscript>
                <img
                  alt="game2"
                  className="cursor-pointer"
                  data-nimg="intrinsic"
                  decoding="async"
                  loading="lazy"
                  src="/_next/image?url=%2Fimages%2Fgame2.png&w=640&q=75"
                  srcSet="/_next/image?url=%2Fimages%2Fgame2.png&w=384&q=75 1x, /_next/image?url=%2Fimages%2Fgame2.png&w=640&q=75 2x"
                  style={
                    Object {
                      "border": "none",
                      "bottom": 0,
                      "boxSizing": "border-box",
                      "display": "block",
                      "height": 0,
                      "left": 0,
                      "margin": "auto",
                      "maxHeight": "100%",
                      "maxWidth": "100%",
                      "minHeight": "100%",
                      "minWidth": "100%",
                      "objectFit": undefined,
                      "objectPosition": undefined,
                      "padding": 0,
                      "position": "absolute",
                      "right": 0,
                      "top": 0,
                      "width": 0,
                    }
                  }
                />
              </noscript>
            </span>
          </a>
        </div>
        <div
          className="w-4/12"
        >
          <a
            href="/login"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            <span
              style={
                Object {
                  "background": "none",
                  "border": 0,
                  "boxSizing": "border-box",
                  "display": "inline-block",
                  "height": "initial",
                  "margin": 0,
                  "maxWidth": "100%",
                  "opacity": 1,
                  "overflow": "hidden",
                  "padding": 0,
                  "position": "relative",
                  "width": "initial",
                }
              }
            >
              <span
                style={
                  Object {
                    "background": "none",
                    "border": 0,
                    "boxSizing": "border-box",
                    "display": "block",
                    "height": "initial",
                    "margin": 0,
                    "maxWidth": "100%",
                    "opacity": 1,
                    "padding": 0,
                    "width": "initial",
                  }
                }
              >
                <img
                  alt=""
                  aria-hidden={true}
                  src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27294%27%20height=%27217%27/%3e"
                  style={
                    Object {
                      "background": "none",
                      "border": 0,
                      "display": "block",
                      "height": "initial",
                      "margin": 0,
                      "maxWidth": "100%",
                      "opacity": 1,
                      "padding": 0,
                      "width": "initial",
                    }
                  }
                />
              </span>
              <img
                alt="game3"
                className="cursor-pointer"
                data-nimg="intrinsic"
                decoding="async"
                src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                style={
                  Object {
                    "border": "none",
                    "bottom": 0,
                    "boxSizing": "border-box",
                    "display": "block",
                    "height": 0,
                    "left": 0,
                    "margin": "auto",
                    "maxHeight": "100%",
                    "maxWidth": "100%",
                    "minHeight": "100%",
                    "minWidth": "100%",
                    "objectFit": undefined,
                    "objectPosition": undefined,
                    "padding": 0,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                    "width": 0,
                  }
                }
              />
              <noscript>
                <img
                  alt="game3"
                  className="cursor-pointer"
                  data-nimg="intrinsic"
                  decoding="async"
                  loading="lazy"
                  src="/_next/image?url=%2Fimages%2Fgame3.png&w=640&q=75"
                  srcSet="/_next/image?url=%2Fimages%2Fgame3.png&w=384&q=75 1x, /_next/image?url=%2Fimages%2Fgame3.png&w=640&q=75 2x"
                  style={
                    Object {
                      "border": "none",
                      "bottom": 0,
                      "boxSizing": "border-box",
                      "display": "block",
                      "height": 0,
                      "left": 0,
                      "margin": "auto",
                      "maxHeight": "100%",
                      "maxWidth": "100%",
                      "minHeight": "100%",
                      "minWidth": "100%",
                      "objectFit": undefined,
                      "objectPosition": undefined,
                      "padding": 0,
                      "position": "absolute",
                      "right": 0,
                      "top": 0,
                      "width": 0,
                    }
                  }
                />
              </noscript>
            </span>
          </a>
        </div>
      </div>
    </div>
  </section>,
  <section
    className="bg-section bg-cover pt-96 pb-20"
    id="about"
  >
    <div
      className="w-8/12 container"
    >
      <div
        className=""
      >
        <h2
          className="xl:text-4xl lg:text-3xl md:text-3xl sm:text-2xl font-body  text-white text-center my-20"
        >
          What's on the impostor website
        </h2>
      </div>
      <div
        className="py-10"
      >
        <h2
          className="xl:text-4xl lg:text-3xl md:text-3xl sm:text-2xl font-body  text-white"
        >
          ABOUT US
        </h2>
        <p
          className="leading-relaxed text-white mt-6"
        >
          Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
        </p>
      </div>
      <div
        className="py-10"
      >
        <h2
          className="xl:text-4xl lg:text-3xl md:text-3xl sm:text-2xl font-body  text-white"
        >
          FREE ONLINE GAMES
        </h2>
        <p
          className="leading-relaxed text-white mt-6"
        >
          Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia, looked up one of the more obscure Latin words, consectetur, from a Lorem Ipsum passage, and going through the cites of the word in classical literature, discovered the undoubtable source.
        </p>
      </div>
      <div
        className="py-10"
      >
        <h2
          className="xl:text-4xl lg:text-3xl md:text-3xl sm:text-2xl font-body  text-white"
        >
          HOW TO PLAY
        </h2>
        <p
          className="leading-relaxed text-white mt-6"
        >
          It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using 'Content here, content here', making it look like readable English. Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model text, and a search for 'lorem ipsum' will uncover many web sites still in their infancy.
        </p>
      </div>
    </div>
  </section>,
  <section
    className="py-20 font-body"
  >
    <div
      className="container"
    >
      <div
        className="flex xl:items-center lg:items-center"
      >
        <div
          className="flex flex-col w-4/12 justify-start px-16"
        >
          <div
            className="flex"
          >
            <span
              style={
                Object {
                  "background": "none",
                  "border": 0,
                  "boxSizing": "border-box",
                  "display": "inline-block",
                  "height": "initial",
                  "margin": 0,
                  "maxWidth": "100%",
                  "opacity": 1,
                  "overflow": "hidden",
                  "padding": 0,
                  "position": "relative",
                  "width": "initial",
                }
              }
            >
              <span
                style={
                  Object {
                    "background": "none",
                    "border": 0,
                    "boxSizing": "border-box",
                    "display": "block",
                    "height": "initial",
                    "margin": 0,
                    "maxWidth": "100%",
                    "opacity": 1,
                    "padding": 0,
                    "width": "initial",
                  }
                }
              >
                <img
                  alt=""
                  aria-hidden={true}
                  src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2730%27%20height=%2730%27/%3e"
                  style={
                    Object {
                      "background": "none",
                      "border": 0,
                      "display": "block",
                      "height": "initial",
                      "margin": 0,
                      "maxWidth": "100%",
                      "opacity": 1,
                      "padding": 0,
                      "width": "initial",
                    }
                  }
                />
              </span>
              <img
                alt="impostor-logo"
                data-nimg="intrinsic"
                decoding="async"
                src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                style={
                  Object {
                    "border": "none",
                    "bottom": 0,
                    "boxSizing": "border-box",
                    "display": "block",
                    "height": 0,
                    "left": 0,
                    "margin": "auto",
                    "maxHeight": "100%",
                    "maxWidth": "100%",
                    "minHeight": "100%",
                    "minWidth": "100%",
                    "objectFit": undefined,
                    "objectPosition": undefined,
                    "padding": 0,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                    "width": 0,
                  }
                }
              />
              <noscript>
                <img
                  alt="impostor-logo"
                  data-nimg="intrinsic"
                  decoding="async"
                  loading="lazy"
                  src="/_next/image?url=%2Fimages%2Fimpostor-logo.svg&w=64&q=75"
                  srcSet="/_next/image?url=%2Fimages%2Fimpostor-logo.svg&w=32&q=75 1x, /_next/image?url=%2Fimages%2Fimpostor-logo.svg&w=64&q=75 2x"
                  style={
                    Object {
                      "border": "none",
                      "bottom": 0,
                      "boxSizing": "border-box",
                      "display": "block",
                      "height": 0,
                      "left": 0,
                      "margin": "auto",
                      "maxHeight": "100%",
                      "maxWidth": "100%",
                      "minHeight": "100%",
                      "minWidth": "100%",
                      "objectFit": undefined,
                      "objectPosition": undefined,
                      "padding": 0,
                      "position": "absolute",
                      "right": 0,
                      "top": 0,
                      "width": 0,
                    }
                  }
                />
              </noscript>
            </span>
            <span
              className="font-bold xl:px-2.5 lg:px-2.5 md:px-0 xl:text-4xl lg:text-4xl md:text-3xl text-primary"
            >
              Impostor
            </span>
          </div>
          <p
            className="mt-5 leading-relaxed"
          >
            Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.
          </p>
        </div>
        <div
          className="w-4/12 px-16"
        >
          <h3
            className="xl:text-3xl lg:text-2xl"
          >
            CONTRIBUTOR
          </h3>
          <ul
            className="mt-5"
          >
            <li>
              <a
                className="text-lg"
                href="#"
              >
                Bogiant
              </a>
            </li>
            <li>
              <a
                className="text-lg"
                href="#"
              >
                Fikri
              </a>
            </li>
            <li>
              <a
                className="text-lg"
                href="#"
              >
                Hasby
              </a>
            </li>
            <li>
              <a
                className="text-lg"
                href="#"
              >
                Oja
              </a>
            </li>
            <li>
              <a
                className="text-lg"
                href="#"
              >
                Ulfa
              </a>
            </li>
          </ul>
        </div>
        <div
          className="xl:w-4/12 lg:w-4/12 md:w-[45%] px-16"
        >
          <h3
            className="xl:text-3xl lg:text-2xl md:text-2xl sm:text-2xl"
          >
            Newsletter
          </h3>
          <form
            className="mt-5"
          >
            <input
              className="w-full border rounded-full py-3 xl:px-10 lg:px-7 md:px-2 sm:px-2"
              name="email"
              placeholder="Enter email address"
              type="text"
            />
          </form>
          <a
            className="mt-3 rounded-full bg-primary text-white text-lg px-10 py-3 inline-block text-center"
            href="/#"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            SUBSCRIBE
          </a>
          <p
            className="mt-5"
          >
            We never spam you!
          </p>
        </div>
      </div>
    </div>
  </section>,
  <footer>
    <div
      className="container text-center py-10"
    >
      <p>
        ©2022 Impostor Team. All Rights Reserved
      </p>
    </div>
  </footer>,
]
`;
